app/components/finance/npq/participant_outcomes/table.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Finance
+  module NPQ
+    module ParticipantOutcomes
+      class Table < BaseComponent
+        attr_reader :participant_declaration
+
+        delegate :outcomes, to: :participant_declaration
+
+        def initialize(participant_declaration:)
+          @participant_declaration = participant_declaration
+        end
+
+        def render?
+          participant_declaration.participant_profile.npq?
+        end
+
+        def completion_date(outcome)
+          if outcome.voided?
+            outcome.updated_at.to_s(:govuk)
+          else
+            outcome.completion_date.to_s(:govuk)
+          end
+        end
+
+        def sent_to_tra(outcome)
+          if outcome.failed?
```

Comment:
I wonder if we should only use `sent_to_qualified_teachers_api_at` as an indication, as failed outcomes can be sent if they are correcting a previous error 🤔

Reply:
@cwrw 
I've based on this around the acceptance criteria on the ticket. Other than that, I agree with you.

Reply:
hmm I mean as long as they are aware that this is the case and want it this way then that's fine